### ISSUE TITLE
BUG: Crash on server startup; only this mod is present

### ISSUE THREAD

**Issue by Daedalus11069** [Lightman314/LightmansCurrency#67]
Minecraft Version: 1.18.2
Forge Version: 40.1.51
Lightman's Currency Version: 1.18.2-1.1.5.3
Other Relevant Mods & Version: N/A
Connection Type: Server

**Describe the bug**
Attempting to run the server with the mod in the server's mods folder causes the server to crash.

**To Reproduce**
1. Create a forge 40.1.51 server.
2. Upload the 1.18.2-1.1.5.3 mod to the mods folder of the server
3. Attempt to run the server(after of course accepting the EULA)

**Expected behavior**
The server does not crash and will run.

**Screenshots/Logs/Crash Reports**
[crash-2022-06-27_20.05.09-fml.txt](https://github.com/Lightman314/LightmansCurrency/files/8997371/crash-2022-06-27_20.05.09-fml.txt)

**Additional context**
Instead of reading the entire crash log above, here's the three lines that I think are important:

```log
Caused by 0: java.lang.NoClassDefFoundError: net/minecraft/client/gui/GuiComponent
		at io.github.lightman314.lightmanscurrency.common.atm.ATMIconData.init(ATMIconData.java:32) ~[lightmanscurrency-1.18.2-1.1.5.3.jar%2338!/:1.18.2-1.1.5.3] {re:classloading,pl:runtimedistcleaner:A}
		at io.github.lightman314.lightmanscurrency.LightmansCurrency.doCommonStuff(LightmansCurrency.java:216) ~[lightmanscurrency-1.18.2-1.1.5.3.jar%2338!/:1.18.2-1.1.5.3] {re:classloading}
```

**Comment by vizthex123** [comment]
Same issue here, though I have other mods installed

Hope it gets fixed soon, guess I'll attach my log if it helps the dev.

[latest.log](https://github.com/Lightman314/LightmansCurrency/files/8998423/latest.log)

**Comment by Lightman314** [comment]
I must have missed some client-only code when setting up the ATM data. I'll make sure it's fixed by the next build (which should be out later today).

**Comment by Lightman314** [comment]
Fixed in v1.1.5.4
